Paintroid/src/main/java/org/catrobat/paintroid/FileIO.kt

```diff
@@ -548,9 +548,10 @@ object FileIO {
             inJustDecodeBounds = false
         }
         val scaling = hasEnoughMemory(resolver, bitmapUri, context)
+        val bitmap = enableAlpha(decodeBitmapFromUri(resolver, bitmapUri, options, context))!!
         return BitmapReturnValue(
             null,
-            enableAlpha(decodeBitmapFromUri(resolver, bitmapUri, options, context)),
```

Comment:
I think you can keep this as it is, any specific reason for the separate variable declaration?

Reply:
i think this is more clean and readable.

Reply:
hmm ok, then please remove the `!!` operators ;D